contracts/sei-tester/src/contract.rs

```diff
@@ -65,18 +66,22 @@ pub fn execute(
         ExecuteMsg::Burn {} => burn(deps, env, info),
         ExecuteMsg::ChangeAdmin {} => change_admin(deps, env, info),
         ExecuteMsg::SetMetadata {} => set_metadata(deps, env, info),
-        ExecuteMsg::TestOccIteratorWrite { values } => test_occ_iterator_write(deps, env, info, values),
-        ExecuteMsg::TestOccIteratorRange { start, end } => test_occ_iterator_range(deps, env, info, start, end),
+        ExecuteMsg::TestOccIteratorWrite { values } => {
+            test_occ_iterator_write(deps, env, info, values)
+        }
+        ExecuteMsg::TestOccIteratorRange { start, end } => {
+            test_occ_iterator_range(deps, env, info, start, end)
+        }
         ExecuteMsg::TestOccParallelism { value } => test_occ_parallelism(deps, env, info, value),
```

Comment:
Curious if there are plans to also add tests for new evm functionality (and possibly extend the integration module)?